CMake/cdat_modules/rpy2_external.cmake

```diff
@@ -0,0 +1,11 @@
+# create an external project to install RPY2,
+# and configure and build it
+set(nm RPY2)
+
+# Set PATH and R_HOME to find R
+list(APPEND USR_ENVS
+  "R_HOME=${cdat_EXTERNALS}/lib/R"
```

Comment:
@aashish24 when we enable the options to use system's externals, let's make sure to set this correctly.
@sankhesh could you set this only if 
1- CDAT_BUILD_R is ON and R_HOME is not already set?